src/Symfony/Component/Intl/ResourceBundle/TimezoneBundleInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Intl\ResourceBundle;
+
+/**
+ * Gives access to locale-related ICU data.
+ */
+interface TimezoneBundleInterface extends ResourceBundleInterface
+{
+    /**
+     * Returns the location name (usually a city) for a timezone.
+     *
+     * @param string $locale        The locale to return the name of (e.g. "de_AT").
+     * @param string $displayLocale Optional. The locale to return the name in.
+     *                              Defaults to {@link \Locale::getDefault()}.
+     *
+     * @return string|null The location name for the timezone or NULL if not found.
+     */
+    public function getTimezoneName($timezone, $displayLocale = null);
+
+    /**
+     * Returns the location names (usually cities) for all known timezones.
+     *
+     * @param string $displayLocale Optional. The locale to return the names in.
```

Comment:
@Tobion I'm not sure understand what you are referring to here. AFAICT optional parameters do not include `|null` in the majority Symfony docblocks. The [coding standards](http://symfony.com/doc/current/contributing/code/standards.html) does not mention it explicitly. But perhaps `|null` is the preferred variant nowadays?